src/components/cell-control/choice.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import React, { forwardRef } from 'react';
+import Autocompleter from './autocompleter.jsx';
+import useForwardedRef from '../../hooks/use-forwarded-ref.js';
+
+const Choice = forwardRef(function Choice(props, forwardedRef) {
```

Comment:
I will test this soon. But since it's just a "configuration" of the autocompleter, it will be very light on the tests.